src/views/ErrorHandler.vue

```diff
@@ -61,6 +61,32 @@ export default class ErrorHandler extends Vue {
             return;
         }
 
+        if (this.keyguardResult.message === Errors.Messages.RESET_PASSWORD) {
+            try {
+                const walletInfo = await this.getWalletForThisRequest();
+                if (walletInfo) {
+                    const request: KeyguardClient.ImportRequest = {
+                        appName: this.request.appName,
+                        requestedKeyPaths: [
+                            DEFAULT_KEY_PATH,
+                            ...[...walletInfo.accounts.values()].map( (account) => account.path),
+                        ],
+                        isKeyLost: false,
+                        expectedKeyId: walletInfo.keyId,
+                        wordsOnly: true,
+                    };
+
+                    const client = this.$rpc.createKeyguardClient();
+                    client.import(request);
+
+                    return;
+                }
```

Comment:
I think there should be an `else` case (when wallet is not found (I know, unlikely but possible?), otherwise you can just use `walletInfo!` everywhere and don't need the `if (walletInfo`) check) where the situation is explained in the error message.

Reply:
Agreed, I added the else case and a comment explaining it.